### ISSUE TITLE
CBTF QUIZ # 2

### ISSUE THREAD

**Issue by seonillee** [uofi-stat430/fa20-disc#25]
Is quiz #2 also going to be an open-book quiz?

**Comment by eddelbuettel** [comment]
Yes, we plan for all to be open book.  That simply makes the most sense given the nature of the course.

**Comment by seonillee** [comment]
I am so sorry for this repetitive question but, from my understanding, open book means that we can use our powerpoint slides during the quiz. Right?
Thank you once again for your time!

**Comment by eddelbuettel** [comment]
Yes. 

 [ But those aren't Powerpoint. They are Markdown generated. ]

**Comment by bojiang3** [comment]
@eddelbuettel  Hi Professor Eddelbuettel, I sent a conflict quiz request via CBTF one day before but didn't receive a reply yet. Could you take a look? THank you

**Comment by eddelbuettel** [comment]
Thanks @bojiang3 -- we do not get a heads-up from CBTF on that so I will take a look now.

**Comment by bojiang3** [comment]
@eddelbuettel Just got it. Thanks!
